test-cases/longevity/longevity-aws-custom-d2-workload1-multidc-big.yaml

```diff
@@ -0,0 +1,270 @@
+    scylla-qa-internal/custom_d2/workload1/latte/custom_d2_workload1.rn
+  - >-
+    latte run --tag latte --duration 1100100 --request-timeout 60 --retry-interval '2s,10s'
+    --sampling 5s --threads 28 --connections 2 --concurrency 56 --rate 560 -P offset=0
+    --function custom -P row_count=1100100 -P codes="\"T9F1\"" --consistency=QUORUM
+    scylla-qa-internal/custom_d2/workload1/latte/custom_d2_workload1.rn
+  - >-
+    latte run --tag latte --duration 1100100 --request-timeout 60 --retry-interval '2s,10s'
+    --sampling 5s --threads 28 --connections 2 --concurrency 56 --rate 560 -P offset=0
+    --function custom -P row_count=1100100 -P codes="\"T10F1\"" --consistency=QUORUM
+    scylla-qa-internal/custom_d2/workload1/latte/custom_d2_workload1.rn
+
+  # NOTE: [region-2-5, loaders 1] other loaders write multi-region tables, second half
+  - >-
+    latte run --tag latte --duration 250050 --request-timeout 60 --retry-interval '2s,10s'
+    --sampling 5s --threads 30 --connections 3 --concurrency 30 --rate 1500 -P offset=250050
+    --function custom -P row_count=250050 -P codes="\"T1F1\"" --consistency=QUORUM
+    scylla-qa-internal/custom_d2/workload1/latte/custom_d2_workload1.rn
+  - >-
+    latte run --tag latte --duration 550050 --request-timeout 60 --retry-interval '2s,10s'
+    --sampling 5s --threads 28 --connections 2 --concurrency 56 --rate 560 -P offset=550050
+    --function custom -P row_count=550050 -P codes="\"T2F1\""
+    scylla-qa-internal/custom_d2/workload1/latte/custom_d2_workload1.rn
+  - >-
+    latte run --tag latte --duration 550050 --request-timeout 60 --retry-interval '2s,10s'
+    --sampling 5s --threads 28 --connections 2 --concurrency 56 --rate 560 -P offset=550050
+    --function custom -P row_count=550050 -P codes="\"T3F1\""
+    scylla-qa-internal/custom_d2/workload1/latte/custom_d2_workload1.rn
+  - >-
+    latte run --tag latte --duration 550050 --request-timeout 60 --retry-interval '2s,10s'
+    --sampling 5s --threads 28 --connections 2 --concurrency 56 --rate 560 -P offset=550050
+    --function custom -P row_count=550050 -P codes="\"T4F1\""
+    scylla-qa-internal/custom_d2/workload1/latte/custom_d2_workload1.rn
+
+stress_cmd:
+  # NOTE: [region-1, loaders 1-5] write to main per-dc tables from loaders of dc-1
+  - >-
+    latte run --tag latte --duration 180m --request-timeout 60 --retry-interval '2s,10s'
+    --sampling 5s --threads 30 --connections 3 --concurrency 180 --rate 15000 -P offset=0
+    --function custom -P row_count=50100100 -P codes="\"T12F1\"" -P print_applied_func_names=2
+    scylla-qa-internal/custom_d2/workload1/latte/custom_d2_workload1.rn
+  - >-
+    latte run --tag latte --duration 180m --request-timeout 60 --retry-interval '2s,10s'
+    --sampling 5s --threads 30 --connections 3 --concurrency 180 --rate 15000 -P offset=0
+    --function custom -P row_count=50100100 -P codes="\"T13F1\"" -P print_applied_func_names=2 --consistency=QUORUM
```

Comment:
why in some command we specify `--consistency=QUORUM` and in some we don't ?

I would guess that the default is `LOCAL_QUORUM`

 I guess there is some  assumption here that one remember by heart codes for queries...

Reply:
Have you read comment on line 139?
It directly influences your question.
First command out of 5 writes to it's own region, other 4 loaders of the first region write to per-dc tables of other 4 regions. Such actiosn requires usage of non-local QUORUM for these `other 4 loaders`.

Reply:
In places where we do not specify `CL` we depend on the default value which is `local_quorum`.

Reply:
I've read the comment, the part about why CL usage need to be different, is not easily deduced from that comment. one need a clear mental picture of the whole setup in mind, and that really hard in this case.

maybe we should consider adding some overall description on qa-internal repo, and point to it.
I'm guessing a diagram for this case might be helpful.